doctr/io/elements.py

```diff
@@ -2,15 +2,18 @@
 
 # This program is licensed under the Apache License version 2.
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+from typing import Any, Dict, List, Optional, Tuple, Union
+from xml.dom import minidom
+from xml.etree import ElementTree as ET
+from xml.etree.ElementTree import Element as ETElement, SubElement
 
-import numpy as np
 import matplotlib.pyplot as plt
-from typing import Tuple, Dict, List, Any, Optional, Union
-
-from doctr.utils.geometry import resolve_enclosing_bbox, resolve_enclosing_rbbox
-from doctr.utils.visualization import visualize_page, synthesize_page
+import numpy as np
+import doctr
 from doctr.utils.common_types import BoundingBox, RotatedBbox
+from doctr.utils.geometry import resolve_enclosing_bbox, resolve_enclosing_rbbox
 from doctr.utils.repr import NestedObject
+from doctr.utils.visualization import synthesize_page, visualize_page
```

Comment:
for future PRs, please don't change the order of imports as this is unrelated to the PR topic :pray:

Reply:
@fg-mindee 
ah yes i see i think i will disable my auto sort if i work on your lib :eyes:

Reply:
We're considering adding isort or something similar but for now, let's not change this :)